OWNERS_ALIASES

```diff
@@ -20,6 +19,7 @@ aliases:
       - slintes
       - mfranczy
       - SchSeba
+      - danielBelenky
```

Comment:
I think this requires a reasoning.
What is the time period you consider an approver to loose his status?

Reply:
According to https://github.com/danielBelenky?tab=overview he has not contributed for a while now. Sure, we should somehow make this official, I guess this is a good point to raise at the community meeting.

Reply:
The main problem I see here is that he get's PR review requests assigned which are not handled. This type of inactivity will putt possible contributors off.

Reply:
Hey @danielBelenky, since this is your approver/reviewer status we are talking about: with regards to this PR, do you still have the time to contribute to this project?

Reply:
> The main problem I see here is that he gets PR review requests assigned which are not handled. This type of inactivity will probably make possible contributors turn away.

Review requests are sent based on the reviewer list, not the approvals. The lists should be distinct.
How about suggesting to remove him from the reviewer list and leaving the approvals intact until a formal procedure is placed (work is under way per what I know).

Reply:
I don't understand why a person that would not review should still be able to approve? Can you elaborate on that please?

Reply:
You have raised the point that review requests are sent automatically to existing reviewers and there is no point to send it if someone is not expected to be actively engaged.
That makes sense and it can be solved by removing from the reviewer list.

The approver, as I see it, is someone who has enough knowledge and experience in a specific project area. I expect this knowledge to exists for some time, and the validity of an approve action to remain valid for a meaningful time.
I do not think this is the place to negotiate what this time should be, but I think 2 months of inactivity is not a meaningful time.

Reply:
OK, convinced, thanks for the explanation!